### PR TITLE
Add recipe for Orange3-Timeseries and Orange3-Network

### PR DESCRIPTION
**Pull request by kernc**
https://pypi.python.org/pypi/Orange3-Timeseries
https://pypi.python.org/pypi/Orange3-Network

Both are hard to compile/install on Windos/OSX.

### CODE REVIEW

**Comment by jakirkham** [comment]
Thoughts on this @astaric?

File: recipes/orange3-timeseries/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+source:
+  fn: Orange3-Timeseries-0.2.5.tar.gz
+  url: https://pypi.python.org/packages/ed/7d/d6b6a2b3ec051eb5a10f5e99b9459a51572f1c4bf3aacd72ba2e9a5a0d77/Orange3-Timeseries-0.2.5.tar.gz
+  md5: 09f3d1c95674683aa53d98df9b539bf5
+  patches:
+    - skip-symlinking-datasets.patch
+
+build:
+  preserve_egg_dir: True
+  number: 1
```
Comment (jakirkham) [inline recipes/orange3-timeseries/meta.yaml]: Please set to `0`.

File: recipes/orange3-timeseries/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package:
+  name: orange3-timeseries
+  version: "0.2.5"
+
+source:
+  fn: Orange3-Timeseries-0.2.5.tar.gz
+  url: https://pypi.python.org/packages/ed/7d/d6b6a2b3ec051eb5a10f5e99b9459a51572f1c4bf3aacd72ba2e9a5a0d77/Orange3-Timeseries-0.2.5.tar.gz
+  md5: 09f3d1c95674683aa53d98df9b539bf5
+  patches:
+    - skip-symlinking-datasets.patch
```
Comment (jakirkham) [inline recipes/orange3-timeseries/meta.yaml]: Could you please elaborate on the need for this?

File: recipes/orange3-timeseries/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package:
+  name: orange3-timeseries
+  version: "0.2.5"
+
+source:
+  fn: Orange3-Timeseries-0.2.5.tar.gz
+  url: https://pypi.python.org/packages/ed/7d/d6b6a2b3ec051eb5a10f5e99b9459a51572f1c4bf3aacd72ba2e9a5a0d77/Orange3-Timeseries-0.2.5.tar.gz
+  md5: 09f3d1c95674683aa53d98df9b539bf5
```
Comment (jakirkham) [inline recipes/orange3-timeseries/meta.yaml]: Please take a look at the `example` recipe included and update this section to [match it]( https://github.com/conda-forge/staged-recipes/blob/0acfeaf41b3cf30c51f3c8de1ce7132f69a8aa90/recipes/example/meta.yaml#L1-L19 ). Please drop any instructive comments the `example` contains.

File: recipes/orange3-timeseries/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  patches:
+    - skip-symlinking-datasets.patch
+
+build:
+  preserve_egg_dir: True
```
Comment (jakirkham) [inline recipes/orange3-timeseries/meta.yaml]: Why is this needed?

File: recipes/orange3-timeseries/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - numpy
+    - scipy >=0.17
+
+test:
+  imports:
+    - orangecontrib.timeseries.tests
+
+about:
+  home: https://github.com/biolab/orange3-timeseries
+  license: AGPLv3+
```
Comment (jakirkham) [inline recipes/orange3-timeseries/meta.yaml]: Please ask upstream to add a license file and included it in `MANIFEST.in` if they have not already.

File: recipes/orange3-timeseries/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  preserve_egg_dir: True
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - statsmodels >=0.6.1
+    - orange3
```
Comment (jakirkham) [inline recipes/orange3-timeseries/meta.yaml]: Are these needed at build time? Are they linked to somehow or merely imported?

File: recipes/orange3-timeseries/meta.yaml (hunk 2 above)
Comment (kernc) [inline recipes/orange3-timeseries/meta.yaml]: Yes, easily. Without it, conda-build fails with:
```
Error: lib/python3.5/site-packages/Orange/datasets/timeseries is a symlink to a path that may not exist after the build is completed (.../conda-bld/orange3-timeseries_1485520571334/work/Orange3-Timeseries-0.2.5/orangecontrib/timeseries/datasets)
```

File: recipes/orange3-timeseries/meta.yaml (hunk 3 above)
Comment (kernc) [inline recipes/orange3-timeseries/meta.yaml]: Shiiit. Using `conda skeleton`, I must have misassumed state of the art output. Will do. Thanks!

File: recipes/orange3-timeseries/meta.yaml (hunk 6 above)
Comment (kernc) [inline recipes/orange3-timeseries/meta.yaml]: Without statsmodels, the recipe wouldn't build locally. I added Orange for the symlink hack in setup.py which assumes the dir is there, before I patched it all away. So it's probably a stale dependency I'll remove.

File: recipes/orange3-timeseries/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+  run:
+    - python
+    - setuptools
```
Comment (astaric) [inline recipes/orange3-timeseries/meta.yaml]: Is setuptools really needed to run the package?

**Comment by kernc** [comment]
@jakirkham, @astaric  do you have perhaps any ideas regarding properly testing these two namespace packages?

**Comment by jakirkham** [comment]
For pure Python packages, [import tests]( https://conda.io/docs/building/meta-yaml.html#python-imports ) are usually sufficient.

**Comment by kernc** [comment]
@jakirkham I thought as much. But as you can see, the [build fails at `import orangecontrib.network`](https://circleci.com/gh/conda-forge/staged-recipes/13059), not willing to find _orangecontrib_, which is a namespace package and the cause of the problem, I assume.

**Comment by kernc** [comment]
Thanks @astaric for debugging and the `skip: True # [py2k]` hack.

**Comment by astaric** [comment]
For the record, the problem was that the tests installed this package in lib/python3.6/site-packages, while everything else got installed into lib/python3.5/site-packages.

My guess is that this behaviour was caused by specifying python version in run/build dependencies.
